hardware_in_the_loop/software/scripts/setup.py

```diff
@@ -0,0 +1,21 @@
+import os
```

Comment:
I'll keep updating this later, don't worry about reviewing it for now

Reply:
might be worth looking into dockerizing this. jack and adi r have a docker system up for the telemetry system